src/legacy/status_im/data_store/chats.cljs

```diff
@@ -59,6 +60,10 @@
            :admins         #{}
            :members-joined #{})))
 
+(defn <-color
+  [value]
+  (if (string/starts-with? value "#") value (keyword value)))
```

Comment:
i think it also makes sense to check against the colors we support in the app if used as a keyword. In status-go there's ~18 customization colors while we support 11.

Reply:
but we set them from the UI, so here we'll have only supported colors

Reply:
btw, do we even need this? the color resolver handles hex strings anyway - that's how community colors are handled

Reply:
https://github.com/status-im/status-mobile/blob/develop/src/quo/foundations/colors.cljs#L342-L346

Reply:
> but we set them from the UI, so here we'll have only supported colors

@flexsurfer i'm thinking of the case when the group chat is created from desktop and the colour is assigned there. If on desktop they have the exact same available colours, then it should be fine i guess.